src/main/java/com/hbm/inventory/recipes/ChemplantRecipes.java

```diff
@@ -369,7 +369,7 @@ public void registerDefaults() {
 				.inputFluids(new FluidStack(Fluids.HEAVYWATER, 8000))
 				.outputFluids(
 						new FluidStack(Fluids.DEUTERIUM, 400),
-						new FluidStack(Fluids.OXYGEN, 400)));
+						new FluidStack(Fluids.OXYGEN, 800)));
```

Comment:
Why _more_ oxygen? Also the O:H ratio for all other recipes has always been 1:1

Reply:
Oh, that's my bad, let me quickly change that back. I got heavy water confused with semiheavy water, since that's what the Girdler-sulfide process seemed to produce.